helm/cluster-apps-operator/values.yaml

```diff
@@ -1,3 +1,11 @@
+appOperator:
+  catalog: control-plane-catalog
+  version: 5.5.1
+
+chartOperator:
+  catalog: default
+  version: 2.20.0
+
```

Comment:
Instead of release CRs this is now how we version app-operator and chart-operator. We can also test feature branches by changing to the test catalog.